### DISCUSSION TITLE
Getting an SSL error?

### DISCUSSION THREAD

**Discussion by nomadsci** [ialbert/bio#29] [Q&A]
I installed bio per the manual onto a miniconda3 env.  But when I try to run bio fetch, I am getting an SSL error.

Example: 

bio fetch AF086833 > refs/AF086833.gb

Results in:  

Error for AF086833, nuccore, gbwithparts, text: <urlopen error [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self signed certificate in certificate chain (_ssl.c:997)>

I tried reinstalling Miniconda3 and bio but still getting the same error.  Any idea where the problem could be?  Thank you!

**Comment by ialbert** [comment]
My first guess would be a misconfigured Python. It is not the `bio` package that has a problem, it is Python that cannot connect at all.

What do you get as output with 

    which python

and

    python -V

**Comment by nomadsci** [comment]
Just add the note, I uninstalled miniconda3 and made a fresh install.   Created a new env called ngs2.  Still get the same error.

which python gives /Users/mark/miniconda3/envs/ngs2/bin/python

Python -v reports the version 3.8.1.

**Reply by nomadsci** [reply]
It must be something specific to my machine.  I was able to run it on a colleague's Macbook without issue.  But not sure how to resolve on my computer as I already tried removing Miniconda3 and making a fresh install...

**Reply by ialbert** [reply]
If you google the error you will see other people experiencing it. My guess would be that your network is using some sort of proxy or VPN to connect and that might be misconfigured.

sometimes it is a security program on the machine that intercepts all networks access for "security" etc.

https://discuss.python.org/t/ssl-certificate-verify-failed-certificate-verify-failed-self-signed-certificate/14359

https://stackoverflow.com/questions/73385267/why-do-i-keep-getting-ssl-certificate-verify-failed-error-when-using-apis
